bobrun/wrap.go

```diff
@@ -100,6 +101,11 @@ func (rw *WithInit) Restart() (err error) {
 func (rw *WithInit) Start() (err error) {
 	defer errz.Recover(&err)
 
+	if rw.inner.Path() != "" {
```

Comment:
This looks the wrong place & method for this. Please don't set a envvar usin `os.Setenv()` rather control it directly through the built in shell. You have control over the environment of the subshell started in `shexec()`  inside this file. The implementation should be more or less similar to this https://github.com/benchkram/bob/blob/main/bobtask/run.go#L28 . At least the part adding the nix dependencies to the `$PATH`. The commander interface should not contain a `Path()` function.